tailwind.config.js

```diff
@@ -244,21 +244,6 @@ module.exports = {
 				90: 90,
 				100: 100,
 			},
-			variants: {
-				rotate: ['hover', 'group-hover'],
```

Comment:
@MarcMathieu we don't need this anymore?

Reply:
Yea all variants are included by default with tailwind v3. So don't need those anymore we can just use them where ever we need